model/database.py

```diff
@@ -0,0 +1,38 @@
+from genericpath import exists
+from re import S
```

Comment:
Why is this import done, I'm little confused?

Reply:
+1